corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -129,7 +129,11 @@ FormplayerFrontend.on("start", function (options) {
         Backbone.history.start();
         // will be the same for every domain. TODO: get domain/username/pass from django
         if (this.getCurrentRoute() === "") {
-            FormplayerFrontend.trigger("apps:list", options.apps);
+            if (options.previewMode) {
+                FormplayerFrontend.trigger("app:preview", options.apps[0]['_id']);
+            } else {
+                FormplayerFrontend.trigger("apps:list", options.apps);
```

Comment:
@wpride i don't think `options.apps` is ever used. i get the sense we're loading all apps and then reloading all apps when we trigger `apps:list`. something to look into, but decided not to do anything about it in this PR